src/dapp/README.md

```diff
@@ -196,10 +196,10 @@ For more reading on property based testing and symbolic execution, see [this tut
 
 While other forms of tests are always run against the post state of the `setUp()` function in the testing contract,
 it can be also be useful to check whether a property is satisfied at every possible contract state. This can be done with
-the `invariant*` testing type. When running an invariant test, hevm will invoke any state mutating function from any 
+the `invariant*` testing type. When running an invariant test, hevm will invoke any state mutating function from any
```

Comment:
```suggestion
the `invariant*` testing type. When running an invariant test, hevm will invoke any state mutating function from all addresses returned by a call to `targetContracts()`, if such a function exists in the testing contracts. If no such method exists, it will invoke methods from any
```

Reply:
Sneaking this change in here :)